modules/ROOT/pages/api-contracts-landing-page.adoc

```diff
@@ -1,15 +1,59 @@
+* xref:external-oauth-2.0-token-validation-policy.adoc[Mule OAuth 2.0 Access Token Enforcement] (Mule v3.8.5 or later)
+* xref:policy-ping-federate.adoc[PingFederate OAuth 2.0 Token Enforcement] (Mule v3.8.5 or later)
+* xref:policy-openid-connect.adoc[OpenID Connect OAuth 2.0 Token Enforcement] (Mule v3.8.5 or later)
+* xref:policy-mule4-jwt-validation.adoc[JWT Validation] (Mule v3.8.5 or later)
+
+For information on how to configure and apply client ID enforcement policies, see xref:client-id-based-policies.adoc[Client ID Enforcement Policy].
+
+== Organization Credentials
+
+Organization credentials provide a method to uniquely identify a specific environment, an organization, or a business group when linking Mule runtime engine (Mule) to an organization using Anypoint Platform. Mule uses these credentials to connect to and access your organization.
+
+For example, you use environment credentials to connect a local Anypoint Studio (Studio) Mule deployment to API Manager through Autodiscovery. This connection allows the Studio-deployed API to be managed by API Manager. See xref:environments-concept.adoc[Review Environment Concepts] and xref:api-auto-discovery-new-concept.adoc[API Autodiscovery].
+
+For additional information on configuring and obtaining organization credentials, see the following:
+
+* xref:org-credentials-config-mule4.adoc[Configure Organization Credentials in Mule Runtime 4].
+* xref:org-credentials-config-mule3.adoc[Configure Organization Credentials in Mule Runtime 3].
+
+== Basic Authentication Policy Credentials
+
+The Simple Authentication default policy protects an API by forcing requests to provide credentials. These credentials populate a request's `Authorization` header. For details on building API requests using the Simple Authentication policy, see xref:basic-authentication-simple-concept.adoc[Basic Authentication: Simple Policy].
+
+The Lightweight Directory Access Protocol (LDAP) Authentication default policy restricts API access by using an LDAP authentication mechanism. LDAP authentication forces API requests to provide credentials in an `Authorization` header, then requests the configured LDAP instance to determine if the credentials are correct in the provided LDAP context. For details on configuring the LDAP Authentication policy, see xref:basic-authentication-ldap-concept.adoc[Basic Authentication: LDAP Policy].
+
+== SOAP API Proxy Credentials
 
-* xref:view-api-contracts.adoc[Viewing API Instance Contracts]
-* xref:manage-client-apps-latest-task.adoc[Granting Access to Requests]
-* xref:remove-client-app-latest-task.adoc[Removing API Instance Contracts]
-* xref:access-client-app-id-task.adoc[Obtaining Client ID and Client Secret Credentials of a Client Application]
\ No newline at end of file
+You can protect a WSDL that is hosted on a remote location by configuring basic authentication credentials. For details on building a SOAP API proxy, see xref:building-soap-proxy.adoc[Build SOAP API Proxies].
```

Comment:
Do we need to spell out WSDL here?

Reply:
Hmmm. My first thought is we shouldn't need to. I'm assuming the audience will already know what a WSDL is.

I searched elsewhere in the API Manager docs and didn't see it spelled out. I know this doesn't necessarily make it okay to _not_ spell it out, but perhaps for consistency we should leave it as is?